rmcreyes/cpen391-user-app
# Admin account view meter status

# :ship: Pull Request

## :question: Purpose

Added views for admin accounts to view meter status
Connected with Parking and Meter APIs
- There are pre-populated data for current and past parking instances just to demonstrate how the data is processed and displayed. Those Will be deleted once we have real data saved in the database

<img src="https://user-images.githubusercontent.com/42983386/110598455-9ef4ea00-8136-11eb-9c80-71c9325d5794.png" width="130" height="300">

## :hammer: Validation Strategy

Manual testing: 
Admin View: 
- Log in with an Admin account (admin@admin.com, password: ADMINADMIN), go to account settings, via Home fragment
- Verify that there is a `View Meter Status` option. 
- Verify that under the `Meter Status` view, the basic info (meter number, if occupied, and unit price) is displayed for each item
- Log out, and log in with an non-admin account (test1@test.com, password: test). 
- Verify that there is not a `View Meter Status` option under `Accounts settings`


Closes #11
